package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": "0.1.0",
+  "name": "com.bluefletch.motorola",
+  "cordova_name": "com.bluefletch.motorola",
+  "description": "Motorola Cordova Plugin",
+  "license": "MIT",
+  "repo": "https://github.com/RsanabriaZ/flq-cordova-plugin-laserscanner.git",
+  "issue": "",
+  "keywords": [],
+  "platforms": [
+    "android"
+  ],
+  "cordova": {
+    "id": "com.bluefletch.motorola",
+    "platforms": [
+      "android"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/RsanabriaZ/flq-cordova-plugin-laserscanner.git"
```

Comment:
Don't use your public repo here. If this repo has been forked over to the foodlogiq organization, then please update this line to use the flq repo.

Reply:
I just copied the package.json content from mine and forgot to change.. I will update it now. Thank you @ryates8